common-react/@dbeaver/ui-kit/src/Spin/Spin.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from 'clsx';
+import './Spin.css';
+import type { ControlSize } from '../types/controls.js';
+
+export interface SpinProps {
+  enabled?: boolean;
+  className?: string;
+  size?: ControlSize;
+}
+
+export function Spin({ className, size, enabled = true }: SpinProps) {
+  return (
```

Comment:
```js
if (!active) {
  return null;
}
```

Reply:
why it's better?

Reply:
compare:
```
return (
  active && (
    ...
  )
);
```

```
return active && (
  ...
);
```


```
if (!active) {
  return null;
}

return (
  ...
);
```

I prefer the last variant, but between the first and second, the first one seems incorrect at all